CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ option(VN_SDK_USE_GHC_FILESYSTEM "Use ghc-filesystem instead of C++ standard <fi
 option(VN_SDK_COVERAGE "Generate gcov statistics for coverage" OFF)
 
 set(VN_SDK_FFMPEG_LIBS_PACKAGE
-    "ffmpeg-libs-minimal"
+    "pkgconfig"
```

Comment:
I'd suggest making this a change to the in `docs/building_linux.md`, rather than changing the default directly. In particular, this change is equivalent to adding `-DVN_SDK_FFMPEG_LIBS_PACKAGE=pkgconfig` to the `cmake -G ..` step

It sounds like you've attempted to build completely without _any_ use of conan, which has never been done before as far as I'm aware. So, any contributions to `building_linux.md` and `getting_started_linux.md` would probably be quite valuable.

Reply:
The reliance of Conan won't work if you want the library distributed as part of the various Linux distros, with these patches, I can compile directly on Fedora 20 using GCC 14.1.1

Reply:
Hm... You may be right that this is a better default.

There will probably be some extremely annoying downstream effects on our CI (which does currently rely on Conan) and our build instructions. But... Technically yeah, probably better...